lib/rules/media-query-no-invalid/README.md

```diff
@@ -62,3 +62,39 @@ The following patterns are _not_ considered problems:
 ```css
 @media (color) and ((hover) or (width)) {}
 ```
+
+## Optional secondary options
+
+### `ignoreFunctions: ["/regex/", /regex/, "string"]`
```

Comment:
This PR isn't directly related, but it might be good to standardize the `ignoreFunctions` docs wording. https://github.com/search?q=repo%3Astylelint%2Fstylelint%20%23%23%23%20%60ignoreFunctions%3A%20%5B&type=code

Reply:
I agree.
I think it should be done at the same time as #3372.